resources/test/heads/test/doctopus-test.edn

```diff
@@ -1,6 +1,6 @@
 {:output-root "docs"
  :html-commands ["lein marg -f index.html"]
  :name "doctopus-test"
- :source-control :git
+ :source-control "git"
```

Comment:
I am A+ on doing that eventually. This was one of the subtle bugs I had to
dig up last night -- and right now, since git is the only DVCS we support,
we aren't actually using that field. I'd say, let's make an issue to
`prepare`/`transform` that, maybe also to wire up, say, mercurial?

On Tue, May 5, 2015 at 10:32 AM, jesse keane <notifications@github.com>
wrote:

> In resources/test/heads/test/doctopus-test.edn
> <https://github.com/Gastove/doctopus/pull/45#discussion_r29692295>:
>
> > @@ -1,6 +1,6 @@
> >  {:output-root "docs"
> >   :html-commands ["lein marg -f index.html"]
> >   :name "doctopus-test"
> > - :source-control :git
> > + :source-control "git"
>
> i had considered possibly doing the prepare / transform dance to make
> these keywords, thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Gastove/doctopus/pull/45/files#r29692295>.
>